blocks/blogheader/blogheader.js

```diff
@@ -0,0 +1,63 @@
+import {
+  getMetadata, decorateIcons, buildBlock, loadBlock, decorateBlock, fetchPlaceholders,
+} from '../../scripts/aem.js';
+import { getLocaleInfo } from '../../scripts/scripts.js';
+import { li } from '../../scripts/dom-helpers.js';
+
+const isDesktop = window.matchMedia('(min-width: 900px)');
```

Comment:
their switch point is `768px`, I think we should use the same.

Reply:
thanks for noting that, changed this to 768